images/cfw/opt/x1plus/share/expansion/expansion.json

```diff
@@ -0,0 +1,54 @@
+{
+  "expansions": {
+    "X1P-002-B": {
+      "name": "X1Plus Expander",
+      "driver": "ft2232",
+      "ports": 2,
+      "image": "x1p-002.png"
+    }
+  },
+  "modules": {
+    "generic_i2c": {
+      "name": "I²C",
+      "configuration": {
+        "i2c": []
```

Comment:
i2c configuration is changed to a list from a dict - should there not be an associated change in i2c.py for a new format, or is this a mistake? Specifically when going over `config.items()` it will fail if the config is stored as a list. 

https://github.com/X1Plus/X1Plus/blob/acf4017d18d907fba0858f9b63dfb6592be29269/images/cfw/opt/x1plus/lib/python/x1plus/services/x1plusd/expansion/i2c.py#L25 

When setting up port_b to generic i2c stemma, it setups the config with the empty list value []. `{"i2c": [], "meta": {"module_config": "generic_i2c"}}`

Manual configuration as dict for config still works, so having the default type here as a list is misleading for configuration unless format changes are needed.

Reply:
For changes - either we change back to a dict to maintain current functionality, or we update i2c.py to reflect it being a list, and document the new format for configuration of multiple i2c devices.

Reply:
I agree, it should be a dict.  Good catch!  Fixed.